### ISSUE TITLE
Following people doesn't add them to the 'People Followed' list

### ISSUE THREAD

**Issue by eternalbit** [mikedilger/gossip#554]
Somehow following people doesn't seem to work.

I'm starting with a fresh key pair without having used nostr before.

I restarted the client multiple times after following people.
The 'People Followed' list just remains empty.

I built gossip from source a few days ago and since it managed a complete build and I can run the application I assumed everything works.

Yes, I added a few relays (9) and set a few to read/write/... and they show as connected.

Whenever I followed a new contact I used the npub & wss option after looking up which relay the person uses.

I started with RUST_LOG="info,gossip::overlord::minion=debug"

Got log messages like:

... INFO src/overlord/mod.rs:938: Setup 1 relay for 8...0
... INFO src/overlord/mod.rs:938: Setup 1 relay for e...9

... DEBUG src/overlord/minion/mod.rs:218: wss://nostr.wine/: Connected
... DEBUG src/overlord/minion/mod.rs:854: NEW SUBSCRIPTION on wss://nostr.wine/ handle=mentions_feed, id=0

The accounts I followed should at least show up in the 'People Followed' list, but it stays at (0).

**Comment by mikedilger** [comment]
Ok let me walk through this. I'm starting fresh.  I get a Welcome to Gossip, Create a New Nostr Account, Generate.  Added minimum relays (3, 2, an 1), continue w/o publishing (since I'm testing), skipping my metadata for now, I'll skip the wizard-follow to see what you are currently seeing. Ok.

I'm going to try fiatjaf.  I entered his key, and pressed follow.  Then I went to the People>Followed page.  He is there.  And moments later his avatar shows up.

I cannot reproduce.

Even if you follow a totally made-up key, the People Followed list should show that key, it should not be at (0).

Please give exact details of how you got there, because I cannot reproduce.

**Comment by mikedilger** [comment]
Also, use the master branch

**Comment by eternalbit** [comment]
I built again, this time using the 'gossip-git' AUR package, to effectively get the master branch.

Following works now.

So the problem was maybe my build then. With first build I used the arch AUR 'gossip' package, which produced version 0.8.1-1.
I chose this package (which uses a tar ball) over the 'gossip-git' package, because the discussion about the package appeared to be more active. Guess I might have to change my heuristic for choosing software :-) : less issues -> less discussion -> pick

Thanks .. and also sorry for wasting your time :| .. I thought building a not so old version would be fine.
